google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/cloudpubsub/PublisherSettings.java

```diff
@@ -87,6 +88,9 @@ public abstract class PublisherSettings {
    */
   abstract boolean enableIdempotence();
 
+  /** Whether to enable request compression. Default true. */
```

Comment:
This is a getter, shouldn't this read as "Whether request compression is enabled."

I'd also fix the wording for enableIdempotence()